ITE/6/README.md

```diff
@@ -0,0 +1,929 @@
+
+*   payloadType “https://in-toto.io/Attestation/v1-json” with a JSON-encoded
+    payload.
+*   payloadType “https://in-toto.io/Attestation/v1-cbor” with a CBOR-encoded
+    payload.
+
+The schema for the payload is described in
+[Claim schema (binding layer)](#claim-schema-binding-layer).
+
+Prior to ITE-5, the existing in-toto signature wrapper may be used. In this
+case, the payload is always JSON with a `_type` of
+“https://in-toto.io/Attestation/v1”.
+
+## Claim schema (binding layer)
+
+A claim is a JSON or CBOR object corresponding to the binding layer. It contains
+the following fields. See subsequent sections for
+[Type definitions](#type-definitions) and [Reasoning](#reasoning).
+
+`attestation_type` _(URI, required)_
+
+> Indicates the meaning of this attestation and how to interpret `details` and
+> `relations`. Example:
+
+```javascript
+"attestation_type": "https://example.com/Provenance/v1"
+```
+
+`subject` _(ArtifactReference, recommended)_
+
+> Identifies what software artifact this attestation is about. Example:
+
+```javascript
+"subject": {"sha256": "728f71…"}
+```
+
+> If unset, the subject is unspecified. This is discouraged and only exists for
+> backwards compatibility and for very rare cases where a single subject does
+> not make sense. Such attestations may not be supported by BinAuthz.
+>
+> Subject always refers to a single artifact (see [reasoning](#reasoning)). If
+> multiple subjects are desired, either: * Output one attestation per subject. *
+> Create a single artifact that encapsulates all other artifacts, such as an
+> archive or hash tree. Then refer to this new artifact as the subject. *
+> (discouraged) Define a custom artifact reference type with whatever semantics
+> are needed.
+
+`relations` _(Map&lt;string, Set&lt;Object>>, optional)_
+
+> Identifies software artifacts related to `subject`. The map key indicates the
+> relationship type and the value contains the set of software artifacts having
+> that relation.
+>
+> Example:
+
+```javascript
+"relations": {
+  "top_level_source": [{
+      "artifact": {"sha256": "a6a63f…"},
+      "filename": "grep_2.12-2.dsc"
+    }],
+  "dependent_sources": [{
+      "artifact": {"sha256": "37887d…"},
+      "filename": "grep_2.12-2.debian.tar.bz2"
+    }, {
+      "artifact": {"sha256": "011998…"},
+      "filename": "grep_2.12.orig.tar.bz2"
+    }],
+  "tools": [{
+      "artifact": {"sha256": "1234…"},
+      "filename": "gcc"
+    }]
+}
+```
+
+> The meaning of the key is dependent on `attestation_type`. Each set value is
+> an _Object_ representing a single software artifact. The schema is dependent
+> on `attestation_type` and can contain arbitrary metadata, but it must always
+> have at least the following field:
+>
+> `artifact` _(ArtifactReference, required)_
+>
+> > Identifies the related software artifact. The following standard relations
+> > are recommended[^1]: * `top_level_source`: The primary input used to
```

Comment:
From @MarkLodato "I'm [...] wondering whether we should have different "classes" of attestations, notably Provenance, CodeReview, TestResult (including vuln scan), and PolicyDecision, each with their own set of relations."

Reply:
We decided against this for now, to keep things simple. We may want to revisit this in a future iteration.